pxtsim/simdriver.ts

```diff
@@ -21,13 +21,24 @@ namespace pxsim {
         nestedEditorSim?: boolean;
         parentOrigin?: string;
         mpRole?: string;    // multiplayer role: "client", "server", or undefined
+        // `messageSimulators` is @DEPRECATED. Use `simulatorExtensions` instead.
```

Comment:
Just curious if the `@` here does something for documentation?

Reply:
Nothing specific, Some syntax coloring algorithms look for jsDoc-like formatting and will visually accentuate it.